pkg/kubelet/cm/cpumanager/topology/topology_test.go

```diff
@@ -1100,3 +1414,38 @@ func TestCPUNUMANodeID(t *testing.T) {
 		})
 	}
 }
+
+func Test_getUncoreCacheID(t *testing.T) {
```

Comment:
I'd like to see tests also for the quite a few helper functions we added, like (in no particular order nor claim of completeness):
- `CPUsPerUncore`
- `UncoreCaches`
- `UncoreInNUMANodes`
- `CPUsInUncoreCaches`
- `CoresNeededInUncoreCache` (not needed if we make it private)

no need to write like 100 test cases, but considering we have a quite rich collection of topologies, we can add few tests for them

Reply:
Quick question then, `getUncoreCacheID` is a private function should I remove this test?

Reply:
Added Unit tests 
- `CPUsPerUncore`
- `UncoreCaches`
- `UncoreInNUMANodes`
- `CPUsInUncoreCaches`

Reply:
> Quick question then, `getUncoreCacheID` is a private function should I remove this test?

let's keep it for now. We can make exceptions, but these should be, well, exceptional :)